deploy/ups-broker/templates/broker.yaml

```diff
@@ -0,0 +1,35 @@
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: ups-broker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: ups-broker
+    spec:
+      containers:
+      - name: ups-broker
+        image: {{ if .Values.registry }}{{ .Values.registry}}/{{ end }}user-broker:{{ if .Values.upsBrokerVersion }}{{ .Values.upsBrokerVersion }}{{ else }}{{ default "latest" .Values.version }}{{ end }}
+        imagePullPolicy: {{ default "Always" .Values.imagePullPolicy }}
+        args:
+        - --port
+        - "8000"
+        ports:
+        - containerPort: 8000
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: ups-broker
+spec:
+{{ if .Values.debug }}
```

Comment:
I see this in the `wip-catalog` char as well. Can you explain why `debug` is the flag that makes a service one of `type: LoadBalancer`. That isn't computing for me.